### PR TITLE
Payee 202 fe library upgrade coronavirus job retention scheme

### PR DESCRIPTION
**Pull request by Jamie22Butler**
- [x] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [x]  HTML Validator passing
- [x] Browser Accessibility Check
- [ ] Reviewed

### CODE REVIEW

File: app/views/JrsClaimSummaryView.scala.html [hunk 1]
```diff
@@ -22,73 +22,79 @@
...
 
 @employerMessageKey = @{viewModel.employerMessageKey}
 
-@mainTemplate(
-    title = messages("check.jrs.claims.title")
+
+@main(
+    title = messages("check.jrs.claims.title"),
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    backLinkUrl = None,
```
Comment (brendanstraughan) [inline app/views/JrsClaimSummaryView.scala.html:37]: What happens when the user clicks to go back if we havent got a URL?

Comment (Jamie22Butler) [reply app/views/JrsClaimSummaryView.scala.html:37]: This page is unique in the case that the back link is handled differently, on lines 48 - 55. Whether we want to handle this in a uniform way, that's up for discussion; however, I did not want to make changes which could affect the layout.
The user navigates back to the whatDoYouWantToDoPage.

So there is a back button, it's just not handled by the main template :)

Comment (brendanstraughan) [reply app/views/JrsClaimSummaryView.scala.html:37]: how come we cant have that back link in here? e.g. 
backLinkUrl = routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage.url,
backLinkContent = Messages("check.jrs.back.link")

Comment (Jamie22Butler) [reply app/views/JrsClaimSummaryView.scala.html:37]: Spoken with Vivek, opted to use the backLink generated from the mainTemplate. 

removed the HTML within the View, as both of them would generate two back links on the page.

**Review by brendanstraughan — APPROVED** [review]

**Pull request merged**